lib/models/models.dart

```diff
@@ -151,6 +149,39 @@ class BalanceModel {
   int get hashCode => super.hashCode;
 }
 
+class FiatRateModel {
+  final Map<String, double> ratesPerUSD;
+  final bool error;
```

Comment:
I dont like the error being inside of the model. 

Shouldnt the error belong to the network response?